src/pages/en/guides/images.md

```diff
@@ -118,6 +118,11 @@ After [installing the integration](/en/guides/integrations-guide/image/#installa
 Astro's `<Image />` and `<Picture />` components cannot be used with images in your `public/` folder. Use standard HTML or Markdown image syntax instead.
 :::
 
+:::caution
+Astro's `<Image />` and `<Picture />` components will throw an error if no `alt` text is provided.
+If no an accessible label is required, set `alt=""`.
+:::
+
```

Comment:
Readers may be confused about when no accessible label is required, so, what do you think of making this a bit more explanatory?

```suggestion
:::caution
Astro's `<Image />` and `<Picture />` components will throw an error if no `alt` text is provided. 

If the image is merely decorative (i.e. doesn't contribute to the understanding of the page), set `alt=""` so it is ignored by screen readers.
:::
```